### PR TITLE
link with libnfs instead of loading it dynamic

### PR DESCRIPTION
**Pull request by Rechi**
## Description
Don't load libnfs dynamic, but link against it (shared/static).

## How Has This Been Tested?
compiled for windows and linux

## Types of change
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [x] All new and existing tests passed

### CODE REVIEW

File: cmake/scripts/android/Install.cmake [hunk 1]
```diff
@@ -153,6 +153,7 @@ foreach(lib IN LISTS required_dyload dyload_optional ITEMS Shairplay)
 endforeach()
 add_bundle_file(${ASS_LIBRARY} ${libdir} "")
 add_bundle_file(${BLURAY_LIBRARY} ${libdir} "")
+add_bundle_file(${NFS_LIBRARY} ${libdir} "")
```
Comment (wsnipex) [inline cmake/scripts/android/Install.cmake]: since we link static, why do we need this?

Comment (Rechi) [reply cmake/scripts/android/Install.cmake]: see PR description
> Don't load libnfs dynamic, but link against it (shared/static).

Currently libnfs is build shared in depends.

Comment (wsnipex) [reply cmake/scripts/android/Install.cmake]: I have both a dynamic and static lib, so we build both. In theory cmake should prefer and link the static lib. If not, we should fix this and not even build the dynamic one

**Comment by candrews** [comment]
My apologies if I'm misunderstanding... would it be better to go with https://github.com/xbmc/xbmc/pull/14188 instead?

File: cmake/scripts/android/Install.cmake (hunk 1 above)
Comment (Rechi) [reply cmake/scripts/android/Install.cmake]: I planed to do the depends changes in a seperate PR. Should I do them in this PR @wsnipex?

Comment (wsnipex) [reply cmake/scripts/android/Install.cmake]: yes please

Comment (Rechi) [reply cmake/scripts/android/Install.cmake]: @wsnipex updated

**Pull request merged**